helm-frontend/src/routes/Home.tsx

```diff
@@ -14,7 +15,6 @@ export default function Home() {
   } else if (window.PROJECT_ID === "mmlu") {
     return <MMLULanding />;
   } else {
-    // TODO: better global/default landing page
-    return <LiteLanding />;
+    return <GlobalLanding />;
```

Comment:
`else if (window.PROJECT_ID == "global")` or something, otherwise `helm-server` users will see this when trying to see their own local results.